string[] boxes_id
float32 length
float32 width
float32 height
---
bool success
string status_message
